README.md

```diff
@@ -1,44 +1,375 @@
+All development of VoiceWP happens transparently on Github. [Github Issues](https://github.com/alleyinteractive/voicewp/issues) are used for identifying and discussing bugs and features. Code contributions, whether fixes or enhancements, should be submitted as Pull Requests.
 
-- Download the .zip file of this repo and upload to your WordPress site by navigating to WP Admin and navigating to **Plugins -> Add New**. Select the 'Upload Plugin' button near the top of the top of the screen to upload the .zip file.
+Join us on [Gitter](https://gitter.im/voicewp/Lobby) for general discussions or questions.
 
-### Flash Briefing Skill
+## Documentation
 
-A Briefings post type is created which is intended to be used for a Flash Briefing skill.
+- [Skills](#skills)
+- [Settings](#settings)
+- [Amazon Developer Console](#amazon-developer-console)
+- [Creating your own custom skills](#creating-your-own-custom-skills)
+- [Filter reference](#hookfilter-reference)
+
+### Skills
+
+#### Flash Briefing Skill
+
+A Briefings post type is created which is intended to be used for the Flash Briefing skill.
 
 The endpoint for this will be at:
 `https://yourdomain.com/wp-json/voicewp/v1/skill/briefing`
 
-### Fact/Quote Skills
+#### Fact/Quote Skills
 
-A Skills post type is created for generic skill creation. Out of the box Fact/Quote skills can be created.
+A Skills post type is created for generic skill creation. Out of the box, Fact/Quote skills can be created.
 
 The endpoint for this will be at:
 `https://yourdomain.com/wp-json/voicewp/v1/skill/(post_id)`
 
-### News from your posts Skill
+#### News Skill
 
-This news/content skill will currently read the 5 latest headlines from your regular WordPress posts and allows the listener to choose a post to be read in full.
+This news/content skill will currently read the 5 latest headlines from your WordPress posts and allows the user to choose a post to be read in full.
 
 The endpoint for this will be at:
 `https://yourdomain.com/wp-json/voicewp/v1/skill/news`
 
-## Credit
+### Settings
+
+The main Alexa Skill Settings screen is located at __Tools -> Alexa Skill Settings__
+
+![setting_menu](https://user-images.githubusercontent.com/784167/27144151-c7ff3b2e-50fe-11e7-8043-431090ec8646.png)
+
+The most important field that needs to be filled in for the news functionality to work is the `News skill ID` field. The Skill ID can be retrieved from the Amazon Developer Console:
+
+![skill_id](https://user-images.githubusercontent.com/784167/27146948-11a683cc-5109-11e7-9afc-502b85696cfa.png)
+
+The top of the settings screen provides the ability to set and customize a number of messages used in the skill. The Welcome message is triggered when a user opens your skill, for example 'Alexa, open Developer Quotes'. (This occurs by Alexa sending a 'LanchIntent' to your skill). The welcome message can prompt the user to take a certain action, or give them a brief intro to the skill.
```

Comment:
Welcome is capitalized in the first instance and lowercase in the subsequent ones.